manager/gui/src/components/alerts/LrAlert.constant.ts

```diff
@@ -87,7 +88,12 @@ export const lrAlertConst = {
         align: "middle",
       },
       { text: "State", value: "state", width: "125px" },
-      { text: "Description", value: "description", type: "custom" },
+      {
+        text: "Description",
+        value: "description",
+        type: "custom",
+        width: "200px",
```

Comment:
you need to increase width for description column.

Reply:
Added a fixed column width to the action column, so that the description column can expand to the maximum available width in the table.